### ISSUE TITLE
Modal content has transparent body content

### ISSUE THREAD

**Issue by miluoshi** [vuejs-tips/vuex-cheatsheet#3]
Content in a modal has transparent body content. Could be solved by setting `background-color: #fff;` to an iframe
See screenshot:
![imgur](https://i.imgur.com/l9PcKuJ.png)

**Comment by mkarnicki** [comment]
I confirm this is an issue.

**Comment by MikeJWms** [comment]
+1

**Comment by hodler-buidler** [comment]
Yeah, the cheatsheet is useful but this bug really spoils the experience of interaction with it. Fix it pls.

**Comment by juliomotol** [comment]
2 years after, no one is still merged the PR.
